src/main/java/liquibase/ext/spanner/VarcharTypeSpanner.java

```diff
@@ -31,7 +31,7 @@ public boolean supports(Database database) {
     @Override
     public DatabaseDataType toDatabaseDataType(Database database) {
         if (database instanceof CloudSpanner) {
-            return new DatabaseDataType("STRING");
+            return new DatabaseDataType("STRING(MAX)");
```

Comment:
You can call `LiquibaseDataType#getParameters()` to get the actual length, and fill that in here instead of always using `MAX`.